### ISSUE TITLE
[ bug ] Details missing after reporting errors

### ISSUE THREAD

**Issue by yonas** [adrienverge/yamllint#758]
Although [super-linter](/super-linter/super-linter) passes `-c parsable`, the line number or a detailed explanation is not given:

```sh
2025-06-18 23:48:12 [INFO]   Linting YAML_PRETTIER items...
  Error: -18 23:48:13 [ERROR]   Found errors when linting YAML_PRETTIER. Exit code: 1.
  2025-06-18 23:48:13 [INFO]   Command output for YAML_PRETTIER:
  ------
  Checking formatting...
  ------
  2025-06-18 23:48:13 [INFO]   Stderr contents for YAML_PRETTIER:
  ------
  [warn] .github/workflows/anchore.yml
  [warn] Code style issues found in the above file. Run Prettier with --write to fix.
  ------
```

source: https://github.com/yonasBSD/toolkit/actions/runs/15745955690/job/44382077337

config:
```yaml
rules:
  document-end: disable
  document-start:
    level: warning
    present: false
  line-length:
    level: warning
    max: 100
    allow-non-breakable-words: true
    allow-non-breakable-inline-mappings: true
```

**Comment by andrewimeson** [comment]
That output appears to  be from Super-linter using Prettier, not yamllint.
